### DISCUSSION TITLE
How to release the mouse pointer with Macbook Pro

### DISCUSSION THREAD

**Discussion by samuelemeo** [86Box/86Box#4570] [Q&A] [answered]
Hi, how can I release the mouse pointer (CTRL+END) with the Macbook Pro keyboard?

The emulator suggests pressing CTRL+END, but what key is END and how do I get this combination?

Can it be customized?

**Comment by cold-brewed** [comment]
Reposting from [a similar question](https://github.com/86Box/86Box/discussions/4272#discussioncomment-8801272):

> If you're on a laptop keyboard, Ctrl+Fn+Right Arrow will work.
> 
> This is because Fn+Right Arrow natively maps to the end key on macOS. Therefore Ctrl+Fn+Right Arrow will send Ctrl+End to release the mouse.

It's not currently possible to customize, but it is planned for the future.

**Reply by ylluminate** [reply]
Any news on how much closer we are to this?

**Reply by Cacodemon345** [reply, marked as answer]
We already have the customization option in recent nightly builds.

**Reply by ylluminate** [reply]
That's fantastic news @Cacodemon345. Thanks!

**Reply by Alexco500** [reply]
One small issue with this. Binded keys are still forwarded to the VM, e.g. PgnDwn for releasing the mouse will result in a scroll down in the active window inside a running VM.
Is there a way to filter this?

**Reply by Cacodemon345** [reply]
I'm afraid this is not easy to filter.

**Comment by RicardoRamosWorks** [comment]
You can use some software that generates macros

**Comment by jvernet** [comment]
Binded keys do not work very well on Mac, specificly on Laptop.... On Mac I would like to use right command key as VirtualBox, but cannot record this binding on MacOsX... Add a not US keyboard and you are nearly in a nightmare...
It's not even perfect with PC under Linux for wich you need to handle FN+Function keys, as it simustanously change the brightness and perform the binded function....

**Reply by OBattler** [reply]
We are limited to combinations Qt can capture and generate a string for, more may be possible by editing the configuration file. And I'm not even sure how to block keys on Linux.

**Comment by Gorilli09** [comment]
It's Fn + Command + Right
